static/js/containers/UserPage_test.js

```diff
@@ -41,11 +44,23 @@ describe("UserPage", () => {
 
   let userActions = [RECEIVE_GET_USER_PROFILE_SUCCESS, REQUEST_GET_USER_PROFILE];
 
+  let openDialog = () => {
+    return [...document.getElementsByClassName('deletion-confirmation')].find(dialog => (
+      dialog.style["left"] === "0px"
```

Comment:
the material-ui dialog isn't an actual `<dialog>` element. basically, when it isn't being displayed (when the `open` prop is set to false) it sets `left = -10000px`, and when it is displayed it changes that to `0px`. this let's us find the one which is currently being 'shown' so we can get buttons and so on inside of it.